lib/Fhp/Model/SEPAAccount.php

```diff
@@ -20,6 +20,14 @@ class SEPAAccount
     /** @var string|null */
     protected $blz;
 
+    /**
+     * Determines how single lines in transaction descriptions are joined, see #481.
```

Comment:
This should somehow refer to MT940 / the fact that this is about transaction _statements_ for this account.